tools/backups2datalad-update-cron

```diff
@@ -30,17 +28,13 @@ set -x
 eval python -m tools.backups2datalad \
    -l WARNING \
    -J 5 \
-   --target "$ds" \
+   --backup-root "$ds" \
```

Comment:
`ds` is /mnt/backup/dandi/dandisets/ I believe, but `path` within `tools/backups2datalad.cfg.yaml` below has  `path: dandisets` and `path: dandizarrs` assuming it is ` /mnt/backup/dandi/`, so either config or these cron scripts should be adjusted (if you while at it create some `backups2datalad-cron-common` to be sourced by all others to set variables -- would be appreciated)

Reply:
Fixed & done.